docs/resources/supported_operations.md

```diff
@@ -0,0 +1,1316 @@
+            aten::sqrt 
+            aten::square 
+            aten::squeeze 
+            aten::stack                                  Supported in limitted set of patterns
+            aten::sub 
+            aten::sum 
+            aten::tan 
+            aten::tan_ 
+            aten::tanh 
+            aten::tanh_ 
+            aten::tensor 
+            aten::to 
+            aten::topk 
+            aten::transpose 
+            aten::tril 
+            aten::triu 
+            aten::type_as 
+            aten::unbind                                 Supported in limitted set of patterns
+            aten::unfold 
+            aten::unsqueeze 
+            aten::unsqueeze_ 
+            aten::upsample_bicubic2d 
+            aten::upsample_bilinear2d 
+            aten::upsample_linear1d 
+            aten::upsample_nearest1d 
+            aten::upsample_nearest2d 
+            aten::upsample_nearest3d 
+            aten::upsample_trilinear3d 
+            aten::var 
+            aten::var_mean 
+            aten::view 
+            aten::where                                  Supported in limitted set of patterns
+            aten::zeros 
+            aten::zeros_like 
+            prim::Constant                               String and None constant is only supported when used by supported operation
+            prim::device 
+            prim::DictConstruct                          Supported in limitted set of patterns
+            prim::dtype                                  Supported in limitted set of patterns
+            prim::GetAttr 
+            prim::If 
+            prim::is_cuda 
+            prim::ListConstruct                          Supported in limitted set of patterns
+            prim::ListUnpack                             Supported in limitted set of patterns
+            prim::Loop 
+            prim::max                                    Supported in limitted set of patterns
+            prim::min                                    Supported in limitted set of patterns
+            prim::NumToTensor 
+            prim::PythonOp                               Supported only in tracing
+            prim::requires_grad 
+            prim::TupleConstruct                         Supported in limitted set of patterns
+            prim::type
+            torchvision::deform_conv2d
+            torchvision::nms
+            torchvision::roi_align
+            ==========================================  ==========================================================================================
+
+         .. tab-item:: TensorFlow
+
+            Some of TensorFlow operations do not match any OpenVINO operations. Yet, they are still supported by 
+            the conversion API (previously Model Optimizer) and can be used on the constant propagation path. These layers are labeled 
+            with ``Constant propagation`` in the table below:
```

Comment:
Please remove these lines. This is not longer valid since TF FE release. All limitations are described per operation.